src/sass/messaging/messaging.scss

```diff
@@ -26,4 +24,5 @@
   width: 80%;
 }
 
-@import "partials/message-category";
+@import "partials/messaging-thread";
+@import "partials/message-subject";
```

Comment:
The risk with putting them at the top is that we could run into inheritance issues as `messaging.scss` grows. Moved the CSS in `messaging.scss` to `partials/_messaging-app.scss` to avoid that problem.